### PR TITLE
Test coverage for cryptomath

### PR DESCRIPTION
**Pull request by tomato42**
Increase test coverage for cryptomath.py module, remove dead code

---
This change is [Reviewable](https://reviewable.io/reviews/tomato42/tlslite-ng/281)

### CODE REVIEW

File: unit_tests/test_tlslite_utils_cryptomath.py [hunk 1]
```diff
@@ -194,6 +194,16 @@ def test_numBits(self, number):
     def test_numBytes(self, number):
         self.assertEqual(numBytes(number), self.num_bytes(number))
 
+
+class TestPowMod(unittest.TestCase):
+    def test_with_small_numerts(self):
```
Comment (Jakuje) [inline unit_tests/test_tlslite_utils_cryptomath.py]: The `numerts` looks like a typo.

File: tlslite/utils/cryptomath.py [hunk 2]
```diff
@@ -253,8 +253,7 @@ def numBytes(n):
 # **************************************************************************
 
 def getRandomNumber(low, high):
-    if low >= high:
-        raise AssertionError()
+    assert low <= high
```
Comment (Jakuje) [inline tlslite/utils/cryptomath.py]: This should say `low < high` to preserve the same semantics. Does it make sense to generate random, where low == high? If so, the change in semantics should be reflected in the commit message.

Comment (tomato42) [reply tlslite/utils/cryptomath.py]: good catch, fixed

File: unit_tests/test_tlslite_utils_cryptomath.py (hunk 1 above)
Comment (tomato42) [reply unit_tests/test_tlslite_utils_cryptomath.py]: indeed it is, fixed

**Comment by tomato42** [comment]
@Jakuje thanks for the review!

**Pull request merged**